packages/client/src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,298 @@
+import { Container, Typography, Link, Box } from "@mui/material";
+import { Link as RouterLink } from "react-router-dom";
+import { ItemComponent } from "./ItemComponent";
+import { useUserStore } from "@/store";
+import { gameDetails } from "@/utils/gameDetails";
+
+export function Home() {
+  const { isAuth } = useUserStore((store) => store.user);
+
+  const scrollToElement = (id: string) => {
+    const element = document.getElementById(id);
+    element?.scrollIntoView({ behavior: "smooth" });
+  };
+
+  return (
+    <Container maxWidth={false} disableGutters={true}>
+      <Box
+        sx={{
+          backgroundImage: "url(../../../public/images/background.png)",
```

Comment:
Не лучше ли было испортировать изображение и передать его в качестве шаблонной строки?) Так пути не потеряются в процессе бандлинга =-)
```suggestion
import backgroundImg from "../../../public/images/background.png";
// ...
          backgroundImage: `url(${backgroundImg})`,
```